docs/creating_review_app.md

```diff
@@ -38,70 +17,62 @@ CircleCI will match the `review-app-` prefix and either:
 2. Update an existing review app using `update_review_app.sh`
 3. Once CI is complete follow [this step](https://github.com/artsy/force/blob/master/docs/creating_review_app.md#dns-setup) to setup DNS.
 
-> Once you're done working with the review app it (currently) has to be deleted manually
-
-#### Deleting a Review App
-
-Run:
-
-```sh
-yarn delete-review-app <name>
-```
-
-The `name` is the name of the branch, minus the `review-app-` prefix.
-
-#### Manual Steps
-
-##### Manual Build
+### Building on Local
 
 First, make sure `jq` is installed:
 
 ```sh
 brew install jq
 ```
 
-Then launch the script:
+Then launch the script with:
 
 ```sh
-./scripts/build_review_app.sh review-app-name
+./scripts/build_review_app.sh awesome-feature
 ```
 
-##### Manual Update
+The script will save a K8s spec in `hokusai/awesome-feature.yml`
+
+### Accessing the review app.
+
+To access the review app, you must create a DNS name for it. A name ending in `artsy.net` is required for full OAuth flows (for, say, login redirects between Gravity and Force for Auction registration). On Cloudflare, please do:
+
+1. [Login to Cloudflare](https://dash.cloudflare.com/), and navigate to **artsy.net** > **DNS**
+1. Click `+ Add Record`
+1. Change `Type` dropdown to `CNAME`
+1. Under `Name` enter `awesome-feature`
+1. Under `Target` say `nginx-staging.artsy.net`
+1. Hit `Save`
+1. DNS will propagate and after a few minutes the review app will be available via `awesome-feature.artsy.net`
+
+### Updating the review app
 
 If you want to push subsequent changes to the review app you can push a new build to the same tag with the `--overwrite` flag:
 
 ```sh
-hokusai registry push --overwrite --skip-latest --force --tag <name>
+hokusai registry push --overwrite --skip-latest --force --tag awesome-feature
 ```
 
 and you need to redeploy your app:
 
 ```sh
-hokusai review_app deploy <name> <name>
+hokusai review_app deploy awesome-feature awesome-feature
 ```
 
-😇 After your review app is no longer needed please remember to clean up any CNAMEs you've created, and to de-provision the review app itself with `hokusai review_app delete <review-app-name>`
+### Deleting a Review App
 
-For more info on Review App maintenence, [see Hokusai docs](https://github.com/artsy/hokusai/blob/master/docs/Review_Apps.md).
-
-#### DNS Setup
+When you are done using the review app, please delete it and its DNS entry.
 
-Regardless of how you created a review app (CircleCI or manually), `build_review_app.sh` will output a hostname like the following:
+Run:
 
 ```sh
-a99199101d01011e9aff2127c3b176f7-1359163722.us-east-1.elb.amazonaws.com
+yarn delete-review-app awesome-feature
 ```
 
-This is your Review App url, which should support all of the basic features inside of Force.
+And please delete the DNS entry as mentioned above.
```

Comment:
It would be nice to have concrete steps here around deleting the DNS entry.

Reply:
Agreed. Done.